installer/book_keeper.iss

```diff
@@ -0,0 +1,64 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Book Keeper"
+#define MyAppVersion "1.0"
+#define MyAppExeName "book_keeper_window.exe"
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application. Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{EDE3DF98-38A6-40AF-94A8-E86C9F55737D}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
+DefaultDirName={autopf}\BookKeeper
+DisableProgramGroupPage=yes
+; Remove the following line to run in administrative install mode (install for all users.)
+;PrivilegesRequired=lowest
+OutputBaseFilename=book_keeper_installer
+SetupIconFile=c:\Users\megyeri_oliver\projects\paginator\installer\icons8-install-58.ico
+Compression=lzma
+SolidCompression=yes
+WizardStyle=modern
+ArchitecturesInstallIn64BitMode=x64
+
+[Languages]
+Name: "english"; MessagesFile: "compiler:Default.isl"
+
+[Tasks]
+Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"; Flags: unchecked
+
+[Files]
+Source: "C:\Users\megyeri_oliver\projects\paginator\dist\book_keeper_window\{#MyAppExeName}"; DestDir: "{app}"; Flags: ignoreversion
+Source: "C:\Users\megyeri_oliver\projects\paginator\dist\book_keeper_window\*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs createallsubdirs
+; NOTE: Don't use "Flags: ignoreversion" on any shared system files
+
+[Dirs]
+Name: "{%TEMP}\book_keeper"
+
+[Icons]
+Name: "{autoprograms}\{#MyAppName}"; Filename: "{app}\{#MyAppExeName}"
+Name: "{autodesktop}\{#MyAppName}"; Filename: "{app}\{#MyAppExeName}"; Tasks: desktopicon
+
+[Registry]
```

Comment:
Tök jó ez a pont, látom nem a szkriptet használod, hanem maga az installer nyúl bele a regisztry-be. Olyat lehetne, hogy ilyenkor nem csak a Szabolcs kulcsát fixálja meg, hanem a többi, felinstallált hangot is, amit a Szabolcs eredeti helyén talál? Mert akkor a progi minden nyelvet tudna használni ami telepítve van, nem csak a magyart. Így akár lehetne majd később egy olyan is, hogy legödrülő listában a gépen elérhető nyelvek/hangok, az nagyon király lenne! (Persze akkor majd a névdetektálást is az alapján változtatom; más nyelvekben ugye nincsennek ragok mint nálunk, ott egyszerűbb dolgunk lesz)